### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by ThanThanhDuy**

### CODE REVIEW

File: src/pages/login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      auth
+        .signInWithPopup(provider)
+        .then(result => {
+          if (result.user) {
+            if (
+              result.user.email &&
+              result.user.email.includes('@fpt.edu.vn')
+            ) {
+              setUser(result.user)
+              console.log(result.user)
```
Comment (jarvis-isme) [inline src/pages/login/Login.jsx:36]: delete console.log

File: src/pages/login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import CloseIcon from '@mui/icons-material/Close'
+import GoogleIcon from '@mui/icons-material/Google'
+import { auth, provider } from '../../utils/firebase/firebase.utils'
+import Grow from '@mui/material/Grow'
+import Scrum from '../../assets/svg/scrum.svg'
+
+function Login() {
+  const [campus, setCampus] = useState('')
+  const [user, setUser] = useState(null)
+  const [open, setOpen] = React.useState(false)
```
Comment (jarvis-isme) [inline src/pages/login/Login.jsx:18]: react.usestate ?

**Comment by jarvis-isme** [comment]
k

File: src/pages/login/Login.jsx (hunk 1 above)
Comment (ThanThanhDuy) [reply src/pages/login/Login.jsx]: I fixed it. Please check again. Thanks!

File: src/pages/login/Login.jsx (hunk 2 above)
Comment (ThanThanhDuy) [reply src/pages/login/Login.jsx]: I fixed it. Please check again. Thanks!

**Pull request merged**